pkgs/development/libraries/science/math/hypre/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, mpi
+, blas
+, lapack
+, openssh
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hypre";
+  version = "2.28.0";
+  # For version info for PETSc, see petsc/config/BuildSystem/config/packages/hypre.py
```

Comment:
I am not sure what the comment should tell us. This package will receive updates if there are ones nad if petsc needs a specific version we should work with overrides/asserts.